### ISSUE TITLE
Error when selling (price set to 1 (W)ETH)

### ISSUE THREAD

**Issue by elwanm** [maximedrn/opensea-automatic-bulk-upload-and-sale#88]
I am facing a problem when selling my NFTs:

1. I get this message for the quantity although it is lower than the offer.
<img width="60%" src="https://user-images.githubusercontent.com/98734893/152852136-08fa68de-fda5-4623-b871-b95279b612c1.png">

2. The price of the item was set at 1 ETH while my price was 0.025 ETH.
<img width="60%" src="https://user-images.githubusercontent.com/98734893/152852245-386d2810-2d23-4f6c-b2dc-c8462aac32ee.png">

Here is my JSON file:
<img width="60%" src="https://user-images.githubusercontent.com/98734893/152852463-a2c86917-d56a-442a-a8e2-d754cfe2f553.png">

I think there is a problem with my JSON format.

**Comment by maximedrn** [comment]
You have some missing data. Look at the data/ folder for upload and sale. You need to have the same order than the examples.
Edition, dna and compiler can be removed.

**Comment by elwanm** [comment]
Do i have to have Method and Sales Type ? 
i just want to list with my price directly

**Comment by maximedrn** [comment]
Yes, you need to have everything. You just have to let them empty

**Comment by elwanm** [comment]
You are a GENUIS ... IT WOKRED PREFECTLY . Thanks Bro

one last question
is there anyway to bulk edit my JSON to meet the same structure

**Comment by maximedrn** [comment]
You can use Visual Studio Code, you will be able to edit multiple lines at the same time

**Comment by elwanm** [comment]
THANKS
